src/common/types/sqsMessage.ts

```
@@ -93,6 +94,11 @@ export const sqsPayloadSchemas = {
       paymentIntentId: z.string().min(1).optional(),
       isVerifiedIdentity: z.boolean()
     })
+  ),
+  [AvailableSQSFunctions.SendSaleFailedEmail]: createSQSSchema(
+    AvailableSQSFunctions.SendSaleFailedEmail, z.object({
```

ESLint

[error] Insert `⏎···`

(prettier/prettier)